sdf/final-project/technical.md

```diff
@@ -6,124 +6,137 @@ This is intended to be filled out by staff members of DPI Tech Apprenticeship pr
 - [ ] **Commit Quality**: Does the project have regular commits with clear, descriptive messages?
-- [ ] **Issue Tracking and Pull Requests**: Is the project utilizing issue tracking to manage tasks and bugs, and are these issues linked to pull requests (PRs)? (at least once)
-
-### Bonus (max 2)
+- [ ] **Pull Requests**: Does the project employ a clear branching and merging strategy, such as Git Flow, to organize development and feature integration?
+- [ ] **Issues**: Is the project utilizing issue tracking to manage tasks and bugs?
+- [ ] **Linked Issues**: Are these issues linked to pull requests (PRs)? (at least once)
 - [ ] **Project Board**: Does the project utilize a project board (e.g., GitHub Projects or Trello) to manage and prioritize work items?
 - [ ] **Code Review Process**: Is there evidence of a code review process, with pull requests reviewed by peers or mentors before merging, ensuring code quality and collaborative learning?
+- [ ] **Branch Protection**: Are the main branches (e.g., master, main) protected to prevent direct commits and ensure code quality?
 - [ ] **Continuous Integration/Continuous Deployment (CI/CD)**: Has the project implemented CI/CD pipelines (using tools like GitHub Actions) to automate testing and deployment?
 
-### Score (/5):
-
 ### Notes:
 
-## Code Hygiene
-- [ ] **Consistent Indentation**: Is the code indentation consistent throughout the application?
-- [ ] **Clear Naming Conventions**: Are variables, methods, and classes named clearly and descriptively? Names should accurately reflect the purpose or functionality of the entity they represent.
-- [ ] **Casing Conventions**: Does the code follow language-specific casing conventions? (e.g., snake_case for Ruby variables and methods, CamelCase for classes).
-- [ ] **DRY Principles**: Are there examples of adherence to the DRY (Don't Repeat Yourself) principle? (eg queries are abstracted to model methods instead of view templates for reusability/readability)
-- [ ] **Code Clarity**: Is the code easy to read and understand? Look for simple, straightforward implementations and avoid unnecessary complexity. Does the code include inline comments that explain "why" behind non-obvious logic? Over-commenting should be avoided; code should be self-explanatory wherever possible. Unused code should be deleted (not commented out).
-
-### Score (/5):
+## Code Hygiene (max: 7 points)
+- [ ] **Indentation**: Is the code consistently indented throughout the project?
+- [ ] **Naming Conventions**: Are naming conventions (e.g., variable names, method names, class names) clear, consistent, and descriptive?
+- [ ] **Casing Conventions**: Are casing conventions (e.g., camelCase, snake_case, PascalCase) consistent throughout the project?
+- [ ] **Layouts**: Is the code making good use of Rails' layouts to provide consistent and reusable templates for the application's views?
```

Comment:
Perhaps rephrashe? Or, adding a comment that explains that projects should correctly utilize the application.html.erb file?

Reply:
done!